api/swagger-spec/v1.json

```diff
@@ -12909,7 +12909,7 @@
    },
    "v1.CinderVolumeSource": {
     "id": "v1.CinderVolumeSource",
-    "description": "CinderVolumeSource represents a cinder volume resource in Openstack. A Cinder volume must exist before mounting to a container. The volume must also be in the same region as the kubelet.",
+    "description": "CinderVolumeSource represents a cinder volume resource in Openstack. A Cinder volume must exist and be formatted before mounting to a container. The volume must also be in the same region as the kubelet. Cinder volumes support ownership management and SELinux relabeling.",
```

Comment:
Cinder volumes do not have to be formatted. The cinder volume implements its own version of FormatAndMount. Incidentally that should use mount.FormatAndMount